### ISSUE TITLE
Uncaught JsonException: Malformed UTF-8 characters, possibly incorrectly encoded

### ISSUE THREAD

**Issue by konradstrek** [AzimoLabs/apple-sign-in-php-sdk#4]
Hi! I'm using PHP 7.4.3 and trying to verify identity token. The response I get: ``Uncaught JsonException: Malformed UTF-8 characters, possibly incorrectly encoded in /home/x/domains/domain.com/public_html/apple/lcobucci/jwt/src/Parsing/Decoder.php:50``. Changing to older PHP version helps, but is there any solution to make it working on >7.4.3? Thanks

**Comment by hywak** [comment]
Hey @konradstrek. 

Thank you for opening this issue. 

To be honest I haven't tested it on version 7.4.3. I will take a look at the issue you posted and prepare a fix in the soon.

Thanks.

**Comment by hywak** [comment]
@konradstrek which version of the library did you use?

I tried to reproduce that issue on `php 7.4.14` using the newest version of the library (`1.2.0`) and there was no error. When I used the older lib version (`1.1.0`) then I got an error but not JSON encoding one.

I suggest using the newest lib version. Ideally, clone clean repo or make sure that `vendor` dir and `composer.lock` file are removed before installing composer dependencies.
